style.css

```diff
@@ -1,4 +1,158 @@
 * {
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
 }
+header{
+  display: flex;
+  justify-content: space-around;
+  position: fixed;
+  z-index: 1;
+  padding: 40px 0;
+  width:100%;  
+}
+.navLink:hover {
+color: #ff9907;
+}
+.homeNavLink {
+  color: #ff9907;
+}
+nav {
+  width: 25%;
+}
+nav ul  {
+  font-family: 'Lora', serif;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between; 
+}
+nav ul li a {
+  text-decoration: none;
+  font-weight: bold;
+  color: #3a414e;
+  font-size: 15px;
+  padding:15px;
+}
+.social-media-containers{
+  width: 100px;
+  margin-right: 83px;
+  display: block;
+}
+.icons {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size :15px;
+  color: #222222;
+  padding: 10px;
+}
+.welcoming-container {
+  position: relative;
+  }
+  .first-picture {  
+    max-width: 100%;
+    height: auto;
+    padding: 75px 120px;
+    filter: brightness(80%);  
+    margin: 22px;   
```

Comment:
the margin here is not necessary, if you notice it causes a horizontal scroll bar on your page, so if you remove it will be better for the responsive design.
![image](https://user-images.githubusercontent.com/54964739/95963396-513ab680-0e10-11eb-807e-1d318666bc0e.png)

Reply:
Thank you for alerting me ! I was looking for a way to get red of that horizontal scroll bar by the way !!